linkerd/app/Cargo.toml

```diff
@@ -32,6 +32,6 @@ regex = "1"
 thiserror = "2"
 tokio = { version = "1", features = ["rt"] }
 tokio-stream = { version = "0.1", features = ["time", "sync"] }
-tonic = { version = "0.10", default-features = false, features = ["prost"] }
+tonic = { workspace = true, default-features = false, features = ["prost"] }
```

Comment:
Not for this change, but we should consider moving the `default-features`/`features` up to the workspace level in the future, IIRC it helps with feature unification and can speed up incremental builds.

Reply:
that's a very good call. i would be on board with that!